### PR TITLE
[WL - 910 (sub branch)] - refactor

### PR DESCRIPTION
**Pull request by dev-tronghieu**
- Widget feature
- Setting feature
- Navigator
- modals
- state
- utils

### CODE REVIEW

File: apps/mobile/src/components/TabsHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import TabItem from './TabItem';
+
+interface Props {
+	style?: ViewStyle;
+	items: TabAble[];
+	activeItem: TabAble;
+	onTabPress?: (item: TabAble) => void;
+}
+
+export const TabsHeader: FC<Props> = ({
```
Comment (tanlethanh) [inline apps/mobile/src/components/TabsHeader/index.tsx:16]: this component is used just in widget feature, not put into `components`

File: apps/mobile/tsconfig.json [hunk 2]
```diff
@@ -1,40 +1,26 @@
...
-        "./browser/bridge/*"
-      ]
+		"paths": {
+			"assets": ["./assets"],
+			"assets/*": ["./assets/*"],
+			"utils": ["./src/utils"],
+			"utils/*": ["./src/utils/*"],
+			"state": ["./src/state"],
+			"state/*": ["./src/state/*"],
+			"components": ["./src/components"],
```
Comment (fluctlight-kayaba) [inline apps/mobile/tsconfig.json:14]: should not use direct dependency this way

File: apps/mobile/src/components/FeatureButtons.tsx/FeatureButton.tsx
Comment (fluctlight-kayaba) [inline apps/mobile/src/components/FeatureButtons.tsx/FeatureButton.tsx:1]: double check path name

File: apps/mobile/src/components/FeatureButtons/FeatureButton.tsx
Comment (fluctlight-kayaba) [inline apps/mobile/src/components/FeatureButtons/FeatureButton.tsx:1]: change name of the file into e.g `Item.tsx` or `Button.tsx`

File: apps/mobile/src/components/FeatureButtons/FeatureButton.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	container: {
+		alignItems: 'center',
+	},
+	innerContainer: {
+		borderRadius: 12,
+	},
+	title: {
+		color: '#4e5e6b',
+		fontSize: 13,
+		fontWeight: '400',
```
Comment (tanlethanh) [inline apps/mobile/src/components/FeatureButtons/FeatureButton.tsx:53]: the default value is '400', remove this line

File: apps/mobile/src/components/FeatureButtons/index.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+			)}
+			{onReceivePress && (
+				<FeatureButton
+					title="Receive"
+					size={buttonSize}
+					onPress={onReceivePress}
+				>
+					<ArrowBottomRight size={iconSize} />
+				</FeatureButton>
+			)}
```
Comment (tanlethanh) [inline apps/mobile/src/components/FeatureButtons/index.tsx:37]: I saw the FeatureButton already has disable style when not passing the onPress, so keep it consistent, not use conditional render

File: apps/mobile/src/modals/Error.tsx [hunk 5]
```diff
@@ -3,13 +3,24 @@ import type { ViewStyle } from 'react-native';
...
+	BindDirections,
+	modalActions,
+	Text,
+	View,
+} from '@walless/gui';
+
+import { ModalId } from './internal';
 
 interface Props {
 	config: ModalConfigs;
```
Comment (tanlethanh) [inline apps/mobile/src/modals/Error.tsx:17]: use ModalConfigs<ErrorModalContext> for correct type check

File: apps/mobile/src/utils/system.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+		Clipboard.setString(value);
+	} else {
+		await navigator.clipboard.writeText(value);
+	}
+
+	showNotificationModal({
+		id: 'copy',
+		prefix,
+		message: 'Copied',
+	});
```
Comment (tanlethanh) [inline apps/mobile/src/utils/system.ts:18]: should not use showModal in utils, prefer separating it, the copy method just takes a copy, and showNotificationModal in which calling this method

**Pull request merged**